components/server/src/iam/iam-session-app.ts

```diff
@@ -42,23 +43,32 @@ export class IamSessionApp {
     }
 
     protected async doCreateSession(req: express.Request) {
-        // We need an account to sign in, which is done by calling `req.login` below.
-        // As proper account creation/selection will be added in later on, we're creating a
-        // dummy account on each attempt.
-        //
-        const user = await this.userService.createUser({
-            identity: {
-                authId: "fake-id-" + Date.now(),
-                authName: "FakeUser",
-                authProviderId: "oidc1",
-                primaryEmail: "fake@email.io",
-            },
-            userUpdate: (user) => {
-                user.name = "FakeUser";
-                user.fullName = "Fake User";
-                user.avatarUrl = "https://github.com/github.png";
+        if (!OIDCCreateSessionPayload.is(req.body)) {
+            throw new Error("Unexpected payload.");
```

Comment:
Invalid argument error?

Reply:
that will be wrapped into a code 500 in the handler.

Reply:
that's true.

Reply:
done